api/test/zero/tools/workloads.clj

```diff
@@ -39,6 +39,7 @@
   "An aou arrays sample for testing."
   {:cromwell      "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org/",
    :notifications [{:chip_well_barcode           "7991775143_R01C01",
+                    :call_rate_threshold         0.98
```

Comment:
Ah! I just added this in my last PR as well (it's on line 45) so you can remove this one

Reply:
Oops, I just noticed it!